modules/machineset-manually-scaling.adoc

```diff
@@ -28,6 +28,28 @@ $ oc get machinesets -n openshift-machine-api
 +
 The machine sets are listed in the form of `<clusterid>-worker-<aws-region-az>`.
 
+. View the machines that are in the cluster:
++
+[source,terminal]
+----
+$ oc get machine -n openshift-machine-api
+----
+
+. Set the annotation on the machine that needs to be deleted:
++
+[source,terminal]
+----
+$ oc annotate machine/<machine_name> -n openshift-machine-api machine.openshift.io/cluster-api-delete-machine="true"
+----
+
+. Cordon and drain the node that needs to be deleted:
```

Comment:
s/that needs to be deleted/that you want to delete:

The original was in passive voice